### PR TITLE
Feature ledstrip brightness + blinker

### PR DESCRIPTION
**Pull request by peterpoetzi**

### CODE REVIEW

File: main/presets.h [hunk 1]
```diff
@@ -229,7 +229,8 @@ constexpr Settings::Ledstrip defaultLedstrip {
     .ledsCount = LEDSTRIP_LENGTH,
     .centerOffset = 1,
     .smallOffset = 4,
-    .bigOffset = 10
+    .bigOffset = 10,
+    .deziampere = 30
```
Comment (CommanderRedYT) [inline main/presets.h:233]: If I flash my boardcomputer, will the LED brightness still be the same or do I have to (massively) increase this value?

File: main/buttons.h [hunk 2]
```diff
@@ -133,5 +137,56 @@ class InputDispatcher
...
+    static void blinkRightButton(bool pressed){
+        if(!pressed)return;
+#ifdef FEATURE_LEDSTRIP
+
+        #ifdef LEDSTRIP_WRONG_DIRECTION
+        const auto target_blinkAnimation = 2;
+#else
+        const auto target_blinkAnimation = 1;
+#endif
+        if(blinkAnimation == 0){ //transition from off to left
```
Comment (CommanderRedYT) [inline main/buttons.h:172]: Two times off to left?
(151 & 172)

**Review by CommanderRedYT — COMMENTED** [review]
Can you move the left/right correction into **main/ledstrip.h**? Now it just corrects for the buttons, but I think we should correct them for the menu and remote control.

File: main/presets.h (hunk 1 above)
Comment (peterpoetzi) [reply main/presets.h:233]: I would increase the brightness to maximum and only change the deziampere setting

File: main/buttons.h (hunk 2 above)
Comment (peterpoetzi) [reply main/buttons.h:172]: ok the comment is wrong

**Comment by peterpoetzi** [comment]
> Can you move the left/right correction into **main/ledstrip.h**? Now it just corrects for the buttons, but I think we should correct them for the menu and remote control.

Yes

**Review by CommanderRedYT — APPROVED** [review]

**Review by CommanderRedYT — APPROVED** [review]

**Pull request merged**